changelog/10953.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+secret/aws: add IAM tagging support for iam_user roles
```

Comment:
```suggestion
secrets/aws: add IAM tagging support for iam_user roles
```

Reply:
I believe this is "more" consistent with other entries. There are in fact some entries that are `secret/` but most I believe are `secrets/`